src/main/java/appeng/container/slot/OptionalSlotFake.java

```diff
@@ -69,12 +69,7 @@ public boolean isSlotEnabled()
 		return this.host.isSlotEnabled( this.groupNum );
 	}
 
-	public boolean renderDisabled()
-	{
-		return this.isRenderDisabled();
-	}
-
-	private boolean isRenderDisabled()
+	public boolean isRenderDisabled()
```

Comment:
Not sure how much it would affect it, but a negated boolean getter is usually strange.